openvpn-install.sh

```diff
@@ -664,57 +679,69 @@ function installOpenVPN () {
 		rm -rf /etc/openvpn/easy-rsa/
 	fi
 
-	# Install the latest version of easy-rsa from source
-	local version="3.0.6"
-	wget -O ~/EasyRSA-unix-v${version}.tgz https://github.com/OpenVPN/easy-rsa/releases/download/v${version}/EasyRSA-unix-v${version}.tgz
-	tar xzf ~/EasyRSA-unix-v${version}.tgz -C ~/
-	mv ~/EasyRSA-v${version} /etc/openvpn/easy-rsa
-	chown -R root:root /etc/openvpn/easy-rsa/
-	rm -f ~/EasyRSA-unix-v${version}.tgz
+	# Install the latest version of easy-rsa from source, if not already
+	# installed.
+	if [[ ! -d /etc/openvpn/easy-rsa/ ]]; then
```

Comment:
This is ineffective because we delete the folder right before :)

Reply:
I suggested to @jmeubank to move the deletion code of easy-rsa that comes with openvpn to be after the openvpn installation to avoid renaming to easy-rsa-auto https://github.com/angristan/openvpn-install/pull/592#pullrequestreview-383455081

@angristan You are right, I missed the fact that we didn't delete the old remove code so now easy-rsa gets removed every time anyway.
https://github.com/angristan/openvpn-install/blob/5805cc03b2a45fb9dd859b60a5e6fe30d2cf28a2/openvpn-install.sh#L677-L680
This should be removed.

Reply:
Yeah. I think I screwed that up with my rebase. Fixing!